Build fails on platforms without sys/cdefs.h

sys/cdefs.h is a non standard non Posix header and is not available eg under Musl libc, but the openlibm version included here includes it unconditionally, in src/openlibm_complex.h and elsewhere

This is fixed in upstream openlibm, see https://github.com/JuliaLang/openlibm/blob/master/src/cdefs-compat.h

On a Musl libc system build fails with 

```
### stderr ###
# ...[truncated]
#                  from ../src/openlibm.h:20,
#                  from libm-test.c:125,
#                  from test-double.c:34:
# ../src/openlibm_complex.h:21:23: fatal error: sys/cdefs.h: No such file or directory
#  #include <sys/cdefs.h>
#                        ^
# compilation terminated.
# make[2]: *** [test-double] Error 1
```

--- a/vars.sh
+++ b/vars.sh
@@ -6,6 +6,6 @@ if [ "${PREFIX}" = "" ]; then
   fi
 fi
 
-LIBM=openlibm-0.4.1-tal1
-LIBM_ARCHIVE=v0.4.1-tal1.tar.gz
-LIBM_URL=https://github.com/talex5/openlibm/archive/${LIBM_ARCHIVE}
+LIBM=openlibm-0.5.4
+LIBM_ARCHIVE=v0.5.4.tar.gz
+LIBM_URL=https://github.com/JuliaLang/openlibm/archive/${LIBM_ARCHIVE}

